src/microbe_stage/IEngulfable.cs

```diff
@@ -0,0 +1,88 @@
+﻿using System.Collections.Generic;
+
+/// <summary>
+///   The ordered process of phagocytosis.
+/// </summary>
+public enum PhagocytosisProcess
```

Comment:
I think this should be put in a separate file, and named PhagocytosisProcessStep / PhagocytosisProcessPhase or PhagocytosisProcessStatus